privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/di/PrivacyConfigModule.kt

```diff
@@ -98,7 +98,6 @@ object DatabaseModule {
     @Provides
     fun providePrivacyConfigDatabase(context: Context): PrivacyConfigDatabase {
         return Room.databaseBuilder(context, PrivacyConfigDatabase::class.java, "privacy_config.db")
-            .enableMultiInstanceInvalidation()
```

Comment:
@marcosholgado I think we don't need it here but wanted to confirm

Reply:
I have doubts, @aitorvs what do you think? I'm not sure if this DB is used with the vpn but we 100% have features in it so maybe we do need it?

Reply:
I would remove this one for now. I think it might be ok but I need to double check carefully.
Also I would not merge this just yet...if you know what I mean ;)

Reply:
Looking for clarity here; @aitorvs your comment could be interpreted either way:

1. Remove the multi-instance invalidation
1. Remove the change (e.g., keep the multi-instance invalidation)

I was assuming you meant we should keep the line in for now (i.e., revert this change). Can you confirm?

Reply:
> Looking for clarity here; @aitorvs your comment could be interpreted either way:
> 
> 1. Remove the multi-instance invalidation
> 2. Remove the change (e.g., keep the multi-instance invalidation)
> 
> I was assuming you meant we should keep the line in for now (i.e., revert this change). Can you confirm?

Lol, I was certainly not clear because I misspelled :D
I meant we should keep the multi-instance invalidation for now (ie. undo the change) for now.

Reply:
Fixed!